lib/engine/game/g_1848/step/special_track.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/special_track'
+
+module Engine
+  module Game
+    module G1848
+      module Step
+        class SpecialTrack < Engine::Step::SpecialTrack
+          def potential_tiles(entity, hex)
+            return @game.tiles.select { |tile| tile.color == 'blue' } if entity.sym == 'P3'
```

Comment:
The tile is in the ability. I don't think you need this special code.

Reply:
yeah, you're right. It wasn't working because tile_valid_for_phase was returning false.
I removed this and added the method in games.rb

Reply:
Okay. The diff is still showing the special code. Was it not removed or am I looking at an old copy?

Reply:
i removed reference to the file in game.rb, but not the actual file. just submitted it